src/handler/bin/index.ts

```diff
@@ -83,23 +91,47 @@ void (async () => {
 		const prefix = data?.prefix ?? '?';
 		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
 		const locale = data?.locale ?? 'en';
-		const lexer = new Lexer(message.content).setQuotes([
-			['"', '"'],
-			['“', '”'],
-			['「', '」'],
-		]);
+		const lexer = new Lexer(message.content).setQuotes(quotes);
 		const res = lexer.lexCommand((s) => (s.startsWith(prefix) ? prefix.length : null));
-		if (!res) continue;
-		const [cmd, tokens] = res;
-		const parser = new Parser(tokens()).setUnorderedStrategy(prefixedStrategy(['--', '-'], ['=', ':']));
-		const out = parser.parse();
-
-		const command = commands.get(cmd.value);
-		if (!command) continue;
-		try {
-			await command.execute(message, new Args(out), locale);
-		} catch (error) {
-			void rest.post(`/channels/${message.channel_id}/messages`, { content: error.message });
+
+		if (res) {
+			const [cmd, tokens] = res;
+			const parser = new Parser(tokens()).setUnorderedStrategy(prefixedStrategy(flagPrefixes, optionSeparators));
+			const out = parser.parse();
+
+			const command = commands.get(cmd.value);
+			if (command) {
+				try {
+					await command.execute(message, new Args(out), locale);
+				} catch (error) {
+					void rest.post(`/channels/${message.channel_id}/messages`, { content: error.message });
+				}
+
+				continue;
+			}
+		}
+
+		{
```

Comment:
is there a missing else here or..? never seen this before

Reply:
Opens a new scope, avoid having to think of names for re-parsing. I don't agree in that avoiding it would "make more sense" per se. However the current approach is unlikely to be introduced anyways, hence the conversion back to draft,